<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddChanDialog</class>
 <widget class="QDialog" name="AddChanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add channel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Master</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="demodTypeCombo">
     <item>
      <property name="text">
       <string>Audio (FM)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Audio (AM)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Audio (USB)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Audio (LSB)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Raw IQ</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Cut-off frequency</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="FrequencySpinBox" name="bandwidthSpinBox"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Frequency</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="masterLabel">
     <property name="text">
      <string>Invalid</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="FrequencySpinBox" name="frequencySpinBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Demodulator type</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Output sample rate</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QRadioButton" name="decimationRadio">
        <property name="text">
         <string>Decimation</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="decimationSpin"/>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="manualRadio">
        <property name="text">
         <string>Manual</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="decimatedRateLabel">
        <property name="text">
         <string>xxx Msps</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="FrequencySpinBox" name="manualRateSpin"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="sourceRateLabel">
        <property name="text">
         <string>xxx Msps</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Native sample rate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrequencySpinBox</class>
   <extends>QWidget</extends>
   <header>FrequencySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddChanDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddChanDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
